assets/js/components/settings/SettingsActiveModule/Footer.js

```diff
@@ -178,27 +177,26 @@ export default function Footer( props ) {
 	let secondaryColumn = null;
 
 	if ( isEditing || isSaving ) {
-		const closeButton = (
-			<Button onClick={ handleClose }>
-				{ __( 'Close', 'google-site-kit' ) }
-			</Button>
-		);
-		const submitButton = (
-			<Button
-				disabled={ isSaving || ! canSubmitChanges }
-				onClick={ handleConfirm }
-			>
-				{ isSaving
-					? __( 'Saving…', 'google-site-kit' )
-					: __( 'Confirm Changes', 'google-site-kit' ) }
-			</Button>
-		);
-
 		primaryColumn = (
 			<Fragment>
-				{ hasSettings && moduleConnected ? submitButton : closeButton }
-
-				<Spinner isSaving={ isSaving } />
+				{ hasSettings && moduleConnected ? (
+					<SpinnerButton
+						disabled={ isSaving || ! canSubmitChanges }
+						onClick={ handleConfirm }
+						isSaving={ isSaving }
+					>
+						{ isSaving
+							? __( 'Saving…', 'google-site-kit' )
+							: __( 'Confirm Changes', 'google-site-kit' ) }
+					</SpinnerButton>
+				) : (
+					<SpinnerButton
+						onClick={ handleClose }
+						isSaving={ isSaving }
+					>
+						{ __( 'Close', 'google-site-kit' ) }
+					</SpinnerButton>
+				) }
```

Comment:
This button should remain being a regular button.

Reply:
Whoops, fixed.